### DISCUSSION TITLE
Undefined Property: $dom

### DISCUSSION THREAD

**Discussion by captenmasin** [livewire/livewire#1523] [4. General]
Has anyone encountered this issue?

```
Undefined property: Livewire\Response::$dom
```
I haven't had a chance to dig through my code and start to debug yet (I'm currently at work) but just wondered if anyone had seen this before and know what it is

(Using 2.1.2)

**Comment by zepfietje** [comment]
@captenmasin, I've encountered this issue too.
Running the following commands fixed it for me:
```console
$ php artisan cache:clear
$ php artisan view:clear
```
